lib/usdUfe/ufe/trf/UsdPointInstanceOrientationModifier.cpp

```diff
@@ -16,73 +16,57 @@
 
 #include "UsdPointInstanceOrientationModifier.h"
 
-#include <mayaUsd/base/api.h>
-#include <mayaUsd/ufe/Utils.h>
+#include <usdUfe/base/api.h>
+#include <usdUfe/ufe/Utils.h>
 
 #include <pxr/base/gf/quath.h>
-#include <pxr/base/gf/vec3h.h>
+#include <pxr/base/gf/rotation.h>
+#include <pxr/base/gf/vec3d.h>
 #include <pxr/pxr.h>
 #include <pxr/usd/usd/attribute.h>
 #include <pxr/usd/usdGeom/pointInstancer.h>
 
-#include <maya/MAngle.h>
-#include <maya/MEulerRotation.h>
-#include <maya/MTransformationMatrix.h>
 #include <ufe/types.h>
 
-PXR_NAMESPACE_USING_DIRECTIVE
-
 namespace {
 // Map of point instance batches under construction, keyed on instancer path.
-MayaUsd::ufe::UsdPointInstanceOrientationModifier::Batches sBatches;
+UsdUfe::UsdPointInstanceOrientationModifier::Batches sBatches;
 } // namespace
 
-namespace MAYAUSD_NS_DEF {
-namespace ufe {
+namespace USDUFE_NS_DEF {
 
 // Ensure that UsdPointInstanceOrientationModifier is properly setup.
 using base = UsdPointInstanceModifierBase<Ufe::Vector3d, PXR_NS::GfQuath>;
-MAYAUSD_VERIFY_CLASS_BASE(base, UsdPointInstanceOrientationModifier);
-MAYAUSD_VERIFY_CLASS_VIRTUAL_DESTRUCTOR(UsdPointInstanceOrientationModifier);
-MAYAUSD_VERIFY_CLASS_NOT_MOVE_OR_COPY(UsdPointInstanceOrientationModifier);
+USDUFE_VERIFY_CLASS_BASE(base, UsdPointInstanceOrientationModifier);
+USDUFE_VERIFY_CLASS_VIRTUAL_DESTRUCTOR(UsdPointInstanceOrientationModifier);
+USDUFE_VERIFY_CLASS_NOT_MOVE_OR_COPY(UsdPointInstanceOrientationModifier);
 
 /* override */
 GfQuath UsdPointInstanceOrientationModifier::convertValueToUsd(const Ufe::Vector3d& ufeValue) const
 {
-    const MEulerRotation eulerRotation(
-        MAngle(ufeValue.x(), MAngle::kDegrees).asRadians(),
-        MAngle(ufeValue.y(), MAngle::kDegrees).asRadians(),
-        MAngle(ufeValue.z(), MAngle::kDegrees).asRadians());
-
-    MTransformationMatrix transformMatrix(MTransformationMatrix::identity);
-    transformMatrix.rotateTo(eulerRotation);
-
-    double real;
-    double i, j, k;
-    transformMatrix.getRotationQuaternion(i, j, k, real);
-
-    return GfQuath(
-        static_cast<float>(real),
-        static_cast<float>(i),
-        static_cast<float>(j),
-        static_cast<float>(k));
+    // Input vector from Ufe is in degrees.
+    PXR_NS::GfVec3f eulerXYZ(ufeValue.x(), ufeValue.y(), ufeValue.z());
+
+    // Create a rotation (XYZ) from the Euler angles.
+    auto rotD = PXR_NS::GfRotation(GfVec3d::XAxis(), eulerXYZ[0])
+        * PXR_NS::GfRotation(GfVec3d::YAxis(), eulerXYZ[1])
+        * PXR_NS::GfRotation(GfVec3d::ZAxis(), eulerXYZ[2]);
+
+    auto            usdQuaternionD = rotD.GetQuaternion();
+    PXR_NS::GfQuath usdQuatD(rotD.GetQuat());
+    return usdQuatD;
```

Comment:
Use USD API to covert value (instead of Maya API). I verified (using asserts from the tests) that the computed result was the same.